e2e_deployment_files/connectivity_deployment.template.json

```diff
@@ -23,18 +23,35 @@
             "env": {
               "UpstreamProtocol": {
                 "value": "<UpstreamProtocol>"
+              },
+              "experimentalfeatures__enabled": {
+                  "value": "true"
+              },
+              "experimentalfeatures__enableMetrics": {
+                  "value": "true"
               }
-             },
+            },
             "settings": {
               "image": "<Container_Registry>/microsoft/azureiotedge-agent:<Build.BuildNumber>-linux-<Architecture>",
-              "createOptions": ""
+              "createOptions": "{\"HostConfig\": {\"PortBindings\": {\"9600/tcp\": [{\"HostPort\": \"9600\"}]}}}"
```

Comment:
see if ExposedPorts option works for edgeAgent and edgeHub.  https://docs.docker.com/engine/api/v1.40/#operation/ContainerCreate

Reply:
I was going to try this, but from the docker inspects I did earlier it is possible there is more configuration on top of PortBindings and ExposedPorts if passing by createOptions. Once I verify that this works I will check if ExposedPorts works.